tests/bin/git

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -eu
+here=$(dirname "${BASH_SOURCE[0]}")
+PATH=":$PATH:"
+PATH="${PATH/":$here:"/:}"
+export PATH="${PATH:1:-1}"
```

Comment:
No, this just appends the directory to the $PATH again whereas the former code removes it from $PATH so the original git is called instead of the script again.

Reply:
I think it would be easier to safe PATH to ORIG_PATH before and then restore it or use something like ``which -a git`` and execute the second entry.